tests/contrib/httplib/test_httplib.py

```diff
@@ -23,7 +23,7 @@
 
 
 # socket name comes from https://english.stackexchange.com/a/44048
-SOCKET = 'httpbin.org'
+SOCKET = 'httpbin-org.herokuapp.com'
```

Comment:
at some point we may use the `kennethreitz/httpbin` container, instead of relying in an external service